spec/neo4j/core/query_spec.rb

```diff
@@ -109,8 +109,7 @@ class Person
       end
     end
 
-    # %i[uuid neo_id].each do |primary_key|
-      %i[neo_id].each do |primary_key|
+    [:uuid, :neo_id].each do |primary_key|
```

Comment:
I checked the specs passes with both `uuid` and `noe_id` so uncommented above code, if it shouldn't be testing for both let me know and I will remove it.